extensions.json

```
@@ -1192,6 +1192,12 @@
       "download": "https://github.com/paiqo/Databricks-VSCode/releases/download/v0.6.0/databricks-vscode-0.6.0.vsix",
       "version": "0.6.0"
     },
+    {
+      "id": "patbenatar.advanced-new-file",
+      "repository": "https://github.com/patbenatar/vscode-advanced-new-file",
+      "version": "1.2.2",
+      "checkout": "master"
```

This repository has no GitHub Releases or dedicated release tag, so there is no need to try and pin or checkout a specific version.

The best we can do here is publish straight from the default branch.

```suggestion
      "repository": "https://github.com/patbenatar/vscode-advanced-new-file"
```